### PR TITLE
Feature/poc

### PR DESCRIPTION
**Pull request by pengin7384**
1. Make transactions
2. Send transactions
3. Verify
image

Since it is poc codes, please skip feedbacks on design or structure! (Design will apply with a next PR)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1 +1,176 @@
...
+
+  /* Output */
+  console.log(
+      `Running time: ${execTime}\n` +
+      `Number: ${number}\n` +
+      `Error: ${errorCount}\n` +
+      `Verified: ${verifiedCount}\n` +
+      `Unverified: ${unverifiedCount}`,
+  );
+
```
Comment (platfowner) [inline index.js:173]: redundant

**Review by platfowner — APPROVED** [review]
LGTM

File: index.js (hunk 1 above)
Comment (pengin7384) [reply index.js:172]: Thanks! Done

Comment (pengin7384) [reply index.js:172]: Done!

**Comment by pengin7384** [comment]
Merged! Thanks 🙃

**Pull request merged**